randovania/games/dread/logic_database/Ghavoran.txt

```diff
@@ -1508,7 +1508,7 @@ Extra - asset_id: collision_camera_021
           Diagonal Bomb Jump (Intermediate) and Lay Normal Bomb
           Flash Shift and Speed Booster and After Ghavoran - Ammo Recharge Enky Above and Speed Booster Conservation (Intermediate)
           # https://www.youtube.com/watch?v=rQiQbXvvCSA
-          Flash Shift and Climb Sloped Tunnels (Advanced) and Wall Jump (Advanced)
+          Flash Shift and Climb Sloped Surfaces (Advanced) and Wall Jump (Advanced)
```

Comment:
Should the Wall Jump here be advanced? I feel like that's the easy part of this trick.

Reply:
I reduced the Wall Jump to Intermediate.